examples/systemd/occi-server.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description="rOCCI-server HTTPS Service"
+After=network.target
+Requires=occi-server.socket
+
+[Service]
+Type=simple
+User=rocci
+WorkingDirectory=/opt/occi-server/embedded/app/rOCCI-server
+Environment="RAILS_ENV=production"
```

Comment:
Are you sure these env variables should even be here? Especially `SECRET_KEY_BASE` and `HOST_KEY`.

Reply:
Yes, with the current setup, they have to be here. For the final release, `SECRET_KEY_BASE` will be generated for every installation, right now it is just a placeholder.

Reply:
I hope to find a better way but this is enough for early testing.